packages/block-library/src/preformatted/theme.scss

```diff
@@ -1,7 +1,5 @@
 .wp-block-preformatted {
-	pre {
```

Comment:
Hi @zzap thank you for your contribution 👍
In the editor, we have the structure of .wp-block-preformatted with a child pre inside. So this change stops applying this styles in the editor causing a design change.
Should we apply this change here bug also move this styles to the editor as code block does?

After:
<img width="613" alt="screenshot 2018-12-03 at 13 25 52" src="https://user-images.githubusercontent.com/11271197/49376533-6ec07480-f6ff-11e8-9df4-b85ac3863e3b.png">

Before:
<img width="662" alt="screenshot 2018-12-03 at 13 25 58" src="https://user-images.githubusercontent.com/11271197/49376540-75e78280-f6ff-11e8-90cc-a46ca67072e1.png">

Its possible to notice a font family difference.

Reply:
Probably easiest for themes wishing to override `pre` styles if we have what @zzap did here for theme.css and we do something like this for the editor:
```
pre { 
 	font-family: inherit; 
 	font-size: inherit; 
 	color: inherit; 
}